lib/send_sms.rb

```diff
@@ -0,0 +1,20 @@
+require 'twilio-ruby'
+require 'time'
+class Messenger
+  def initialize()
+    @account_sid =
+    @auth_token = ''
+    @client = Twilio::REST::Client.new(@account_sid, @auth_token)
+    @from = '+447445034137' # Your Twilio number
```

Comment:
It's best to avoid putting sensitive info like phone numbers in source code that you upload to the internet as it can easily get into the wrong hands! You can use something called an environment variable to avoid having to do that. https://www.rubyguides.com/2019/01/ruby-environment-variables/